content/shmem_and.tex

```diff
@@ -0,0 +1,44 @@
+\apisummary{
+  Atomically perform a non-fetching bitwise and operation on a
+  remote data object.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+void shmem_and(TYPE *dest, TYPE value, int pe);
+\end{C11synopsis}
+where \TYPE{} is one of the standard \ac{AMO} types specified by
+Table~\ref{stdamotypes}.
+
+\begin{Csynopsis}
+void shmem_<TYPENAME>_and(TYPE *dest, TYPE value, int pe);
+\end{Csynopsis}
+where \TYPE{} is one of the standard \ac{AMO} types and has a corresponding
+\TYPENAME{} specified by Table \ref{stdamotypes}.
+
+\begin{apiarguments}
+
+  \apiargument{OUT}{dest}{A pointer to the remotely accessible data object to
+    be updated.}
+  \apiargument{IN}{value}{The operand to the bitwise atomic operation.}
+  \apiargument{IN}{pe}{An integer value for the \ac{PE} on which \VAR{dest}
```

Comment:
I _can_ change it to "PE number of the remote PE. _pe_ must be of type integer." I'm not going to say my text is much better, but I find the whole argument-description section rather irritating in the OpenSHMEM specification. It generally seems redundant and unnecessary, given a well-written description body. I guess that's a different battle to fight, or at least not one this PR will solve.

So, should this text be consistent with the rest of the AMOs, or with the RMA descriptions?